js/ui/map.js

```diff
@@ -575,7 +575,7 @@ class Map extends Camera {
      * representing features within the specified vector tile or GeoJSON source that satisfy the query parameters.
      *
      * @param {string} sourceID The ID of the vector tile or GeoJSON source to query.
-     * @param {Object} parameters
+     * @param {Object} [parameters]
```

Comment:
I suppose so it is :wink: 

Given `parameters.sourceLayer` and `parameters.filter` were optional I incorrectly assumed I could omit `parameters` entirely but I guess the JS doc says you still need to provide an empty object.